### DISCUSSION TITLE
Hidden beatmaps can be found, navigated to and played

### DISCUSSION THREAD

**Discussion by Pennek** [ppy/osu#16558] [Q&A]
Hidden beatmaps can be found in the track playlist. The audio from a hidden beatmap can also be played in song select through the track playlist even if another song is selected, effectively overriding it.

If the player tries to play a beatmap while the audio is overridden, the beatmap from the hidden track will be played instead, leading to some strange behaviour when the beatmap is quit.  The background and leaderboard will be from the hidden track while the song title card and difficulties will be from the original beatmap.

_(note: this can also be done with osu!'s intro tracks which I'm guessing are handled similarly to hidden tracks)_

![osu_2020-11-26_18-12-39](https://user-images.githubusercontent.com/20502902/100378167-df5eab00-3012-11eb-9c5a-fca571c8748a.jpg)

**video:** https://youtu.be/CIBS6eMoJu8
**osu!lazer version:** 2020.1121.0

**Comment by peppy** [comment]
Similar to https://github.com/ppy/osu/issues/6199 and likely other issues. This is due to using lookups that bypass the soft delete flag in some places.

**Comment by peppy** [comment]
I'm not actually sure this is an issue. It's generally a feature and I think we'll probably keep things this way. Going to convert this to a discussion.

**Comment by y0rune** [comment]
Hello!
Today I accidentally hide the map and I could not find it. I was looking at the discussions and found it (little bit old :D) The above feature/issue still exist on the latest version (Lazer 2023.717.0).
It will be nice if I can unhide or select the hidden map.

**Reply by bdach** [reply]
there's a button in the settings menu for that:

![osu_2023-07-24_00-05-53](https://github.com/ppy/osu/assets/20418176/b11d6107-ac1c-4453-822b-28593fdbc809)

**Reply by y0rune** [reply]
All right! Thank you and sorry for that :D
